fermenter-mda/src/test/java/org/technologybrewery/fermenter/mda/element/ServiceSteps.java

```diff
@@ -280,47 +285,47 @@ public void no_service_metamodel_instance_is_returned_for_the_name_in(String nam
         assertNull("Should not have found a service for this package and name!", loadedService);
```

Comment:
Q: Should these asserts be updated to the new `org.junit.jupiter.api.Assertions` package?